# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  d08fc9bb3b8cc15de28c7f5d44c40547fc3ab079
Misc: Add ! to helpus.end, to see get 'Make a donation!'

diff --git a/mail/locales/en-US/chrome/messenger/aboutDialog.dtd b/mail/locales/en-US/chrome/messenger/aboutDialog.dtd
--- a/mail/locales/en-US/chrome/messenger/aboutDialog.dtd
+++ b/mail/locales/en-US/chrome/messenger/aboutDialog.dtd
@@ -42,17 +42,17 @@
 <!ENTITY community.end3             " working together to keep the Web open, public and accessible to all.">
 
 <!ENTITY helpus.start               "Want to help? ">
 <!-- LOCALIZATION NOTE (helpus.donateLink): This is a link title that links to the thunderbird donation page -->
 <!ENTITY helpus.donateLink          "Make a donation">
 <!ENTITY helpus.middle              " or ">
 <!-- LOCALIZATION NOTE (helpus.getInvolvedLink): This is a link title that links to https://www.thunderbird.net/get-involved/. -->
 <!ENTITY helpus.getInvolvedLink     "get involved!">
-<!ENTITY helpus.end                 "">
+<!ENTITY helpus.end                 "!">
 
 <!ENTITY releaseNotes.link          "Release notes">
 
 <!-- LOCALIZATION NOTE (bottomLinks.license): This is a link title that links to about:license. -->
 <!ENTITY bottomLinks.license        "Licensing Information">
 
 <!-- LOCALIZATION NOTE (bottomLinks.rights): This is a link title that links to about:rights. -->
 <!ENTITY bottomLinks.rights         "End-User Rights">
